README.md

```diff
@@ -1,3 +1,5 @@
+**This branch has been archived at January 2017 and will not be updated.**
+
```

Comment:
please merge to the php-7.0 branch instead of the master branch. The README file in master is not archived.